Example/Backpack/Story.swift

```diff
@@ -18,14 +18,13 @@
 
 import Foundation
 
-protocol Story: RawRepresentable, CaseIterable {
+protocol Story {
   var presentableStory: Presentable { get }
+  var title: String { get }
 }
 
-extension Story where Self.RawValue == String {
-  static var storyStructure: [Item] {
-    return self.allCases.map({ story in
-      return Item(name: story.rawValue, value: .story(story.presentableStory))
-    })
+extension Story {
+  var example: Item {
+    Item(name: self.title, value: .story(self.presentableStory))
```

Comment:
`Story` is now the base protocol for use with a single story item, e.g. the colours story.